src/pages/registration/RegistrationFormActions.tsx

```diff
@@ -75,49 +78,65 @@ export const RegistrationFormActions = ({
         }}>
         {tabNumber > RegistrationTab.Description && (
           <Box sx={{ gridArea: 'back-button' }}>
-            <Button
-              variant="outlined"
-              data-testid="button-previous-tab"
-              startIcon={<ArrowBackIcon />}
-              onClick={() => setTabNumber(tabNumber - 1)}>
-              {tabNumber === RegistrationTab.ResourceType && t('registration.heading.description')}
-              {tabNumber === RegistrationTab.Contributors && t('registration.heading.resource_type')}
-              {tabNumber === RegistrationTab.FilesAndLicenses && t('registration.heading.contributors')}
-            </Button>
+            <Tooltip title={t('common.previous')}>
+              <IconButton onClick={() => setTabNumber(tabNumber - 1)} disabled={isFirstTab}>
+                <Box
+                  sx={{
+                    bgcolor: isFirstTab ? 'gray' : 'info.main',
+                    display: 'flex',
+                    borderRadius: '50%',
+                    padding: '0.2rem',
+                  }}>
+                  <ChevronLeftIcon sx={{ color: 'white' }} />
```

Comment:
Ser det er litt forskjellig hos oss vs i figma nå (venstre er NVA, høyre er Figma) 
![bilde](https://user-images.githubusercontent.com/6313468/219339246-b2143e09-634a-422d-8ad7-dc8fb44a3357.png)

For en eller annen grunn får jeg ikke til å inspecte elementet i Figma heller, så kanskje avklare med Cecilie hvordan hun vil det skal være? (om det ikke er gjort allerede da)

Reply:
nvm, etter litt mer trykking i figma ser jeg at fargen er `primary.light`, og ikonet er `KeyboardArrowLeft`

Reply:
Kan se det ved å sjekker `vegviser`-komponenten